packages/tsconfig.json

```diff
@@ -14,6 +14,7 @@
     { "path": "waci-pex-qr-react" },
     { "path": "bls-kms-local" },
     { "path": "bls-did-provider-key" },
+    { "path": "bls-key-did-resolver" },
```

Comment:
Can we rename it to bls-did-resolver-key, more inline with the providers?

Reply:
Done!